pfio/NetCDF4_FileFormatter.F90

```diff
@@ -287,11 +287,11 @@ subroutine open(this, file, mode, unusable, comm, info, rc)
       else
          status = nf90_open(file, IOR(omode, NF90_SHARE), this%ncid)
       end if
+      !$omp end critical
```

Comment:
```suggestion
      
```
The OpenMP critical section should protect all IO, including the write below

Reply:
Critical constructs need to be named.  Otherwise they are global.   But I doubt OpenMP is related to anything going on in this ticket.

Reply:
I agree with Atanas that the end critical should remain where it was.